### PR TITLE
Lms1xx node diagnostics

### PR DESCRIPTION
**Pull request by destogl**
This is a node with extended diagnostic.

Even though I am thinking to switch anyway to [LMS1xx from clearpathrobotics](https://github.com/clearpathrobotics/LMS1xx)... I will test it next days...

### CODE REVIEW

File: cob_sick_lms1xx/common/include/lms1xx.h [hunk 1]
```diff
@@ -194,112 +194,112 @@ typedef enum {
...
+    void startDevice();
 
 private:
-	bool connected;
-	bool debug;
+    bool connected;
+    bool debug;
 
-	int sockDesc;
+    int sockDesc;
```
Comment (fmessmer) [inline cob_sick_lms1xx/common/include/lms1xx.h]: please keep indentation style as it was...
at least do not mix tabs vs. spaces throughout source files within this package...

**Comment by destogl** [comment]
sorry for so many commits, I tried to squash them not to generate the new ones :(

**Comment by fmessmer** [comment]
it still seems to be a mixture of tabs/spaces indentation....please go for one or the other...**no mixture**

**Comment by destogl** [comment]
Oh I hate it! I always work with spaces (because of python), but than you have few files with tabs :/

What is a standard for cob code?

**Comment by fmessmer** [comment]
as only raw3-5 is using lms1xx and travis is happy, we can merge this now..

**Comment by destogl** [comment]
Thanks!

**Pull request merged**